yoga/Yoga.c

```diff
@@ -698,6 +705,18 @@ YG_NODE_LAYOUT_RESOLVED_PROPERTY_IMPL(float, Margin, margin);
 YG_NODE_LAYOUT_RESOLVED_PROPERTY_IMPL(float, Border, border);
 YG_NODE_LAYOUT_RESOLVED_PROPERTY_IMPL(float, Padding, padding);
 
+void YGNodeStyleSetDisplay(const YGNodeRef node, const YGDisplay paramName) {
+  if (node->style.display != paramName) {
+    node->style.display = paramName;
+    YGNodeMarkDirtyInternal(node);
```

Comment:
no need to call `YGNodeMarkDirtyInternal` here as `YGNodeMarkChildrenDirtyInternal` also marks the passed in node as dirty